### PR TITLE
create a new docker-on-arm.md document

### PR DESCRIPTION
**Pull request by madmas**
I also stumpled over the missing document linked from the main README (#85) and thought we could start a new document and added the first thing I found when started to use docker on raspberry pi.

### CODE REVIEW

File: docs/docker-on-arm.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Please keep in mind that this is intended as a collection of important / useful things that are different when running containers on different processor architectures. 
+It's most certainly not final and if you have some objections, please create a pull-request :-)
+
+## Architecture does matter
+
+Event if many modern software stacks encapsulate differences between architectures and you can easily 
+run a application based on Java as well as one based on NodeJS on a PC (x86) and on a RaspberryPi (ARM), the underlying 
+components still are dependend on the architecture.
+
+So when you build your docker image for your application, you most certainly create a platform dependend image as you either build
```
Comment (luxas) [inline docs/docker-on-arm.md]: depend

File: docs/docker-on-arm.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+It's most certainly not final and if you have some objections, please create a pull-request :-)
+
+## Architecture does matter
+
+Event if many modern software stacks encapsulate differences between architectures and you can easily 
+run a application based on Java as well as one based on NodeJS on a PC (x86) and on a RaspberryPi (ARM), the underlying 
+components still are dependend on the architecture.
+
+So when you build your docker image for your application, you most certainly create a platform dependend image as you either build 
+upon on that already has specific packages included or you include such in your docker file.
```
Comment (luxas) [inline docs/docker-on-arm.md]: `Dockerfile`

**Comment by luxas** [comment]
Thanks @madmas!
I've been on a one-week journey, so answering now.
I found only two nits, then LGTM

I'll probably put some more information there in the future, about `qemu` emulation, the new docker registry manifest format, cross-compiling etc., but thanks for writing this up! :smile: 
(I'm a bit short on time right now)

**Comment by madmas** [comment]
Thanks for the feedback, I fixed the typos :)

**Comment by luxas** [comment]
LGTM

**Comment by luxas** [comment]
BTW, you've targeted this PR to `master` although it should be `dev`
This means that I already merged the change in `dev`, but it will show up in master on the next release

**Pull request merged**